src/Purebred/Types.hs

```diff
@@ -438,6 +448,14 @@ fbSearchPathKeybindings = lens _fbSearchPathKeybindings (\cv x -> cv { _fbSearch
 fbHomePath :: Lens' FileBrowserSettings FilePath
 fbHomePath = lens _fbHomePath (\s a -> s { _fbHomePath = a })
 
+data AddressbookSettings = AddressbookSettings
```

Comment:
[nit] can we capitalise `Book` i.e. `AddressBookSettings`?

Reply:
Yep, done!